certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -3091,38 +3592,123 @@
       }
     },
-    "/api/v1/Individuals/{id}/Customfields": {
-      "post": {
+    "/api/v1/Webhooks/RequestUpdated": {
+      "x-ms-notification-content": {
+        "description": "When customer request is updated.",
+        "schema": {
+          "$ref": "#/definitions/CustomerRequestPayload"
+        }
+      },
+      "post": {
+        "tags": ["Webhook", "Customer"],
         "responses": {
           "200": {
             "description": "Success",
             "schema": {}
           }
         },
-        "summary": "Update Custom Field Value",
-        "description": "Update Custom Field Value.",
-        "operationId": "Update-custom-field-value",
+        "summary": "When customer request is updated",
+        "description": "Triggers when a customer request is added or closed.",
+        "operationId": "When-Request-Updated",
+        "x-ms-trigger": "single",
         "parameters": [
           {
-            "name": "id",
-            "in": "path",
+            "name": "Content-Type",
+            "in": "header",
             "required": true,
             "type": "string",
-            "description": "Customer Id.",
-            "x-ms-summary": "Customer Id",
-            "x-ms-url-encoding": "single"
+            "default": "application/json",
+            "x-ms-visibility": "internal"
           },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "typeName": {
+                  "type": "string",
+                  "description": "Type Name.",
+                  "title": "Type Name",
+                  "x-ms-visibility": "internal",
+                  "default": "Customer.RequestUpdated"
+                },
+                "filters": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "title": "Additional Filters",
+                    "x-ms-visibility": "internal",
+                    "default": "NA"
+                  },
+                  "description": "Filters."
+                },
+                "secret": {
+                  "type": "string",
+                  "description": "Secret.",
+                  "title": "Secret",
+                  "x-ms-visibility": "internal",
+                  "default": "PowerAutomate"
+                },
+                "callBackUrl": {
+                  "type": "string",
+                  "description": "Callback URL.",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": "Callback URL"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Description.",
+                  "title": "Description",
+                  "x-ms-visibility": "internal",
+                  "default": "Added By Power Automate"
```

Comment:
@RjImpx It is not allowed to swagger contain any of the Microsoft Power Platform product names

Documentation - [Define descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)